src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
We are making big changes to this contract. Please note that the shape will change shortly:
- Removing Windows-only APIs (DuplicateAndClose, etc)
- Removing APM APIs and adding TPL equivalents
- Removing some of the constructors that make blocking I/O calls.